README.md

```diff
@@ -370,6 +370,29 @@ Utilize the following method if an end-user requests to have their data deleted:
 SPConsentManager.clearAllData()
 ```
 
+## Set a Property Group Id
+ Property groups allow your organization to group properties together in order to simplify configurations for  mass campaigns and updates. 
+ In order to use a Property Group Id, you should edit the SDK configuration object as follows:
+ 
+ ```swift
+ lazy var consentManager: SPConsentManager = { SPConsentManager(
+    accountId: 22,
+    propertyName: try! SPPropertyName("mobile.multicampaign.demo"),
+    campaigns: SPCampaigns(
+        gdpr: SPCampaign(groupPmId: "123") // <- "123" is the property group id
```

Comment:
~// <- "123" is the property group id~
// <- "123" is the id of the privacy manager for the property group

Reply:
Same is valid for the content below. We should substitute _Property Group Id_ for _Privacy Manager Id for the Property Group_.